src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -239,9 +240,7 @@ void testGetPredefinedShelfNameReturnsCorrectShelf(String shelfName) {
             case "Did not finish":
                 expectedShelf = DID_NOT_FINISH;
         }
-        PredefinedShelf.ShelfName actualShelf =
-                predefinedShelfUtils.getPredefinedShelfName(shelfName);
-        assertThat(actualShelf).isEqualTo(expectedShelf);
+      assertEquals(expectedShelf, predefinedShelfUtils.getPredefinedShelfName(shelfName).orElse(PredefinedShelf.ShelfName.DID_NOT_FINISH));
```

Comment:
A couple of changes are needed here:

- assertThat
- Adhere to the 100 character column limit